nixos/modules/virtualisation/azure-common.nix

```diff
@@ -1,67 +1,88 @@
-{ lib, pkgs, ... }:
+{ config, lib, pkgs, ... }:
 
 with lib;
+let
+  cfg = config.virtualisation.azure;
+  mlxDrivers = [ "mlx4_en" "mlx4_core" "mlx5_core" ];
```

Comment:
Could you let us know what these drivers are for ? Accelerated networking ?

Reply:
Yes, these are added following the official documentation of enabling accelerated networking for custom VM images: https://learn.microsoft.com/en-us/azure/virtual-network/accelerated-networking-overview?tabs=redhat#custom-vm-images